### DISCUSSION TITLE
Different behaviour for built-in date_created and custom field

### DISCUSSION THREAD

**Discussion by eikaramba** [directus/directus#13888] [Archive]
### Describe the Bug

When i use the system default fields for a collection i get a different timezone behaviour as if i "recreate" the field myself. All the settings are the same

### To Reproduce

e.g. create a new collection with date_created field
then create a new entry
the api returns
`{date_created:"2022-06-14T13:23:03.630Z"}`

now delete the field, and recreate it in the same way (setting current time on create)
now the api return

`{date_created:"2022-06-14T13:24:51"}`

**this is happening on the directus cloud**

### What version of Directus are you using?

9.12.1

### What version of Node.js are you using?

?

### What database are you using?

free cloud (sqlite?)

### What browser are you using?

FF

### How are you deploying Directus?

cloud

**Comment by rijkvanzanten** [comment]
Are you using a timestamp or a datetime field?

**Reply by eikaramba** [reply]
datetime field. i have the "built in" field and my own created field side-by-side here on 2 collections. the data model looks the same. however they behave the same. i am also very confused honestly :)

**Reply by eikaramba** [reply]
i can also reproduce the error in the cloud, so it does not seem to be something specific to my setup

**Reply by rijkvanzanten** [reply]
The difference between those two is that one is a timestamp (includes the timezone information `Z`), and the other a datetime 🤔 The "user_created" field is just a regular field that's preset by the app during creation, so there's no special system sauce that can affect how it returns beyond the configuration that the user can also do. If they're both returned marked as a "datetime", that feels odd. Any thoughts on that @licitdev?

**Reply by licitdev** [reply]
Datetime fields do not contain timezone information. It's a fixed "flat" value. In the API responses, timestamp fields contains timezone information ending with `Z`. 
The system creates the `date_created` and `date_updated` as timestamps.
If you were to manually create the `date_created` field, you should be selecting the schema type as `timestamp`.

The schema type can be found within the brackets when hovered, in this case it's `Timestamp`. 
The `Datetime` beside `date_created` is the selected interface.

![image](https://user-images.githubusercontent.com/26413686/173596137-99a9a603-bb84-459f-af97-0e1418114c5c.png)

**Reply by eikaramba** [reply]
that did the Trick! sorry for not seeing this. i will certainly write this down :)
